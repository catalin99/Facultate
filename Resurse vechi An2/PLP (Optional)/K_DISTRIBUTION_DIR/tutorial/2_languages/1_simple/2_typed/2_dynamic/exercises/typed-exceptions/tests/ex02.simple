void main() {
  int e = 100;
  try {
    try {
      int x = 5;
      if (x >= 0) {
        throw x+2;
      }
      print(20, "\n");                      // should not be printed
    } catch(string e) {
      print(e," as a string\n");            // should not be printed
    }
  } catch(int e) {
    print(e, " as an int\n");               // should print 7
  }
  print(e," as in the top declaration\n");  // should print 100
}
