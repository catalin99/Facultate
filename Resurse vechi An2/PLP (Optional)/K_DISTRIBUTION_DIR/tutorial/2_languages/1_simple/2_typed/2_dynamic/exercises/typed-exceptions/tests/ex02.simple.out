7 as an int
100 as in the top declaration
